internal/ignition/ignition.go

```diff
@@ -244,6 +244,7 @@ type IgnitionConfig struct {
 	ServiceCACertPath    string        `envconfig:"SERVICE_CA_CERT_PATH" default:""`
 	ServiceIPs           string        `envconfig:"SERVICE_IPS" default:""`
 	SkipCertVerification bool          `envconfig:"SKIP_CERT_VERIFICATION" default:"false"`
+	EnableOKDSupport     bool          `envconfig:"ENABLE_OKD_SUPPORT" default:"true"`
```

Comment:
Why default it to true?

Reply:
Default it's true to maintain current behaviour in non-SaaS deployments